src/__tests__/stopping.test.ts

```diff
@@ -0,0 +1,739 @@
+import { Cat } from '..';
```

Comment:
do we have tests the cat will stop when item bank is used up?

Reply:
Yes, all lines are covered now

Reply:
where is the test when item bank is used up?

Reply:
line 303 is the test if you want a set number, if you don't add early stopping it will go trough all

Reply:
And line 98 in `src/__tests__/clowder.test.ts` tests that Clowder will return undefined when an item bank for a specified `catToSelect` is used up.